packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -94,6 +116,7 @@ export interface EditGeneratorOpWeights {
 	start?: number;
 	commit?: number;
 	abort?: number;
+	synchronize?: number;
```

Comment:
throughout this file, i think you'd be better served by doing the following:

1. Make all members of this interface required
2. Declare `defaultEditGeneratorOpWeights` with this type
3. In places where you want to "support setting some of these weights, if the user wants to specify them", take in a `Partial<EditGeneratorOpWeights>`. Everywhere else, take in `EditGeneratorOpWeights`.
4. When creating a generator, use the standard JS spread pattern `function makeGenerator(optionsArg: Partial<MyOptionsObject>) { const options = { ...defaultOptions, ...optionsArg }; }

This should simplify a lot of the logic around `?? 0`--you'll be able to compile-time confirm all of these values are set.

Reply:
updated with Partial<EditGeneratorOpWeights>